tests/serial/watch/WatchIntegralScaleAndNoScaleParallel.cpp

```diff
@@ -0,0 +1,140 @@
+#ifndef PRECICE_NO_MPI
+
+#include "testing/Testing.hpp"
+
+#include <precice/precice.hpp>
+#include <vector>
+#include "helpers.hpp"
+#include "io/TXTTableWriter.hpp"
+
+BOOST_AUTO_TEST_SUITE(Integration)
+BOOST_AUTO_TEST_SUITE(Serial)
+PRECICE_TEST_SETUP("SolverOne"_on(1_rank), "SolverTwo"_on(1_rank))
+BOOST_AUTO_TEST_CASE(WatchIntegralScaleAndNoScaleParallel)
+{
+  PRECICE_TEST();
+
+  using Eigen::Vector2d;
+
+  if (context.isNamed("SolverOne")) {
+    precice::Participant interface(context.name, context.config(), 0, 1);
+
+    // Set mesh
+    Vector2d coordA{0.0, 0.0};
+    Vector2d coordB{1.0, 0.0};
+    Vector2d coordC{1.0, 2.0};
```

Comment:
This is the same for both participants. Could you merge this into a single vector/array and use bulk functions setMeshVertices/readData instead?
This would reduce this tests size by a lot.